pup-0002.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Docstring Style Conventions
+author: Austin Macdonald
+created: 28-Mar-2017
+status: proposed
+---
+
+## Summary
+
+This PUP documents docstring style conventions for Pulp 3.0.
+
+## Motivation
+
+To improve docstring readability and increase information density.
+
+## Details
+
+Upon acceptance, these changes will be documented in the Pulp style guide.
+
+### Style Choices
+
+Docstrings should generally follow the conventions provided in the comments section of the
+[Google style guide.](http://google.github.io/styleguide/pyguide.html?showone=Comments#Comments)
+
+Additionally, docstrings must be parsable by the
+[Napoleon plugin for Sphinx.](http://www.sphinx-doc.org/en/stable/ext/napoleon.html)
+
+### Additions/Exceptions
+
+Pulp conventions that differ from the `Comments` section of the Google style guide will be
+explicitly included in the Pulp style guide.
+
+1. Modules will not contain license information. Our license information is presented correctly
+within the project; it is unnecessary to include it in each file.
+
+2. The type of each `Arg` value should be included after the variable name in parentheses. The type of each `Returns` value should be the first item on the line. 
+
+3. Following the type of `Arg` and `Return` values, there will be a colon and a single space
+followed by the description.
+
+### Porting from Pulp 2.Y
+
+The 3.0 development guide will be updated to state that all ported code should contain updated
+docstrings. Regular expressions to ease large changes will be included there.
+
+## Drawbacks:
+
+It will take time to convert each docstrings into the new style.
+
+## Alternatives
+
+Some alternatives that have been discussed include:
+
+1. Tabular style to align types and descriptions.
```

Comment:
Yeah, moving the "tabular"-ness of docstrings to Unresolved Questions is a good call.